yt/fields/fluid_fields.py

```diff
@@ -195,8 +195,8 @@ def grad_func(axi, ax):
         slice_3dl = slice_3d[:axi] + (sl_left,) + slice_3d[axi+1:]
         slice_3dr = slice_3d[:axi] + (sl_right,) + slice_3d[axi+1:]
         def func(field, data):
-            block_reorder = getattr(data, '_block_reorder', 'C')
-            if block_reorder == 'F':
+            reorder = getattr(data, '_gradient_swap_axes', False)
```

Comment:
oh, never mind, I see now -- this is an additional check for what we were already doing.

Reply:
I am actually still confused as to why this is required to start with... any guess?

Reply:
There are some strange corner cases where we get F/C ordering in these that isn't consistent, because of mixed ordering among the axes.

Reply:
I corrected this in 10cbdec, so this should be good now.